Made url and home page empty in new json. Added ignore case for .git directory

TheHollidayInn opened the pull request:
This should fix: https://github.com/openkosmosorg/kubos-sdk/issues/41

Also, I noticed that the new project was copying .git. This was causing issues for me as well as I believe the git should not be copied.

rplauche commented on kubos-sdk.py:
```diff
@@ -224,11 +228,36 @@ def link_mounted_modules(): # Globally link the dev modules to replace the stand
         yotta.main()
 
+#Allows for copying into existing directory for shutil
+def copytree(src, dst, symlinks = False, ignore = None):
```
Can we use shutil.copytree instead of pulling in the implementation?

TheHollidayInn replied:
This is a workaround to allow for copying and overwriting existing directories. Since the command use the project folder name and directory, I had to use this.

TheHollidayInn commented:
Updated. I think this may be a better solution. I removed the copytree function I added. And now, the project_dir variable is used (it wasn't before). So, now to create a new kubos project you can skip the create a dir and cd into it. You simple start with:

`kubos init my-new-project`

And a dir called my-new-project will be created with the default files. Does that work?

marshall commented:
@TheHollidayInn apologies for the delay, we have the CLA ready to sign: https://www.clahub.com/agreements/openkosmosorg/KubOS

marshall commented on kubos-sdk.py:
```diff
@@ -86,12 +88,14 @@ def _init(name):
         sys.exit(1)
 
-    copy_tree(container_source_dir, os.getcwd())
+    shutil.copytree(container_source_dir, proj_name_dir, ignore=shutil.ignore_patterns(".git"))
```
nit: use single quotes for the string here

marshall commented on kubos-sdk.py:
```diff
@@ -29,6 +29,8 @@
 
 from distutils.dir_util import copy_tree
+import shutil
+import stat
```
can you organize these alphabetically with the red of the Python built-in libraries above?

marshall commented:
Review done, this looks good! Can you also submit a documentation update to reflect the new usage and behavior of "kubos new"?

Once we get that this is good to merge!

marshall commented:
Er I meant kubos init :)

kyleparrott commented:
This looks good to me too. So I am merging.
